psalm.xml

```diff
@@ -6,6 +6,8 @@
     xmlns="https://getpsalm.org/schema/config"
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
     errorBaseline="psalm/psalm-baseline.xml"
+    findUnusedBaselineEntry="true"
+    findUnusedCode="false"
```

Comment:
Why `findUnusedCode="false"`?

Reply:
Because `psalm` is not aware of things that happen under the hood of laminas, so e.g. calls to functions from a router are not recorded as function calls meaning no code ever is 'used' according to `psalm`s definitions

Reply:
And I added it because it'll become a requirement once the next major release of psalm is out

Reply:
> And I added it because it'll become a requirement once the next major release of psalm is out

Great!

> Because `psalm` is not aware of things that happen under the hood of laminas, so e.g. calls to functions from a router are not recorded as function calls meaning no code ever is 'used' according to `psalm`s definitions

Stuff like `$this->getRequest->[is|get]Post()`? In the website code we have been adding type hints to make sure that everything understands what happens (PhpStorm also had some issues with inferring types).